hopp/simulation/technologies/pv/pv_plant.py

```diff
@@ -220,6 +254,18 @@ def footprint_area(self):
         area = num_modules * module_attribs['area']
         return  area
 
+    @property
+    def land_coverage_area(self):
+        """Estimate Total Module Land Cover Area [m^2]"""
+        import numpy as np
```

Comment:
I think we should move the import to the top of the file

Reply:
removed property